src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -31,15 +31,27 @@
  */
 final class OrderedClassElementsFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
-    /** @internal */
+    /**
+     * @internal
+     *
+     * @deprecated use `alpha_case_insensitive` instead
+     */
     public const SORT_ALPHA = 'alpha';
 
+    /** @internal */
+    public const SORT_ALPHA_CASE_INSENSITIVE = 'alpha_case_insensitive';
+
+    /** @internal */
+    public const SORT_ALPHA_CASE_SENSITIVE = 'alpha_case_sensitive';
+
```

Comment:
Instead of deprecating `alpha` algorithm IMHO it would be better to introduce new config option `case_insensitive` with boolean as allowed values and default value representing current behavior. This way we don't need to deprecate option which was just introduced and we can handle case sensitiveness for any other sorting algorithm (instead of providing 2 options for each, like you did for `alpha`).

Reply:
Excellent idea, thanks. Done.

PS:
Sorry for disturbing :)

Reply:
Really great that in the end you did `case_sensitive` instead of `case_insensitive`, I meant one but pasted another 😅.

PS. Not a problem 👍.

Reply:
Really great that in the end you did case_sensitive instead of case_insensitive, I meant one but pasted another 😅.

PS. Not a problem 👍.